examples/js/nodes/materials/PhongNode.js

```diff
@@ -131,7 +131,7 @@ THREE.PhongNode.prototype.build = function( builder ) {
 		var normal = this.normal ? this.normal.buildCode( builder, 'v3' ) : undefined;
 		var normalScale = this.normalScale && this.normal ? this.normalScale.buildCode( builder, 'fv1' ) : undefined;
```

Comment:
@sunag Sorry, I do not think I am making myself clear. In your code:

    "vec3 perturbNormal2Arb( vec3 eye_pos, vec3 surf_norm, vec3 map, vec2 mUv, float scale ) {",

`scale` should be a `vec2`, and so should `NormalMapNode.scale`, because `MeshPhongMaterial.normalScale` is a `Vector2()`.

In the examples, `normalScale` often must be `Vector2( 1, - 1 )` -- presumably because of the `flipY` setting.